scripts/prepare_fb_data.py

```diff
@@ -0,0 +1,36 @@
+import pdb
+import csv
+import sys
+
+"""
+Process the historical data from FB groups to conform the
+chatbot-retrieval data format.
+"""
+
+# NOTE: we don't make use of num_likes at all
+def parse_info_from_line(line):
+  raw = line.split("\t")
+  post = {'num_likes': raw[0], 'body': raw[1].rstrip()}
+  raw_comments = raw[2:]
+  comments = []
+  for i in range(0, len(raw_comments), 2):
+    comments.append({'num_likes': raw_comments[i], 'body': raw_comments[i+1].rstrip()})
+  return {'post': post, 'comments': comments}
+
+def process_raw_data(filename, output):
+  with open(filename) as f1:
```

Comment:
Thanks! Could we use gzopen to save disk space? Also, we need encoding, 'utf-8' somewhere. 

See https://github.com/hunkimForks/DeepQA/blob/master/chatbot/fbdata.py